prowler/providers/aws/services/wafv2/wafv2_webacl_logging_enabled/wafv2_webacl_logging_enabled.metadata.json

```diff
@@ -0,0 +1,34 @@
+{
+  "Provider": "aws",
+  "CheckID": "wafv2_webacl_logging_enabled",
+  "CheckTitle": "Check if AWS WAFv2 logging is enabled",
+  "CheckType": [
+    "Logging and Monitoring"
+  ],
+  "ServiceName": "wafv2",
+  "SubServiceName": "",
+  "ResourceIdTemplate": "arn:partition:wafv2:region:account-id:webacl/webacl-id",
+  "Severity": "medium",
+  "ResourceType": "AwsWafv2WebAcl",
+  "Description": "Check if AWS WAFv2 logging is enabled",
+  "Risk": "Enabling AWS WAFv2 logging helps monitor and analyze traffic patterns for enhanced security.",
+  "RelatedUrl": "",
+  "Remediation": {
+    "Code": {
+      "CLI": "aws wafv2 update-web-acl-logging-configuration --scope REGIONAL --web-acl-arn arn:partition:wafv2:region:account-id:webacl/webacl-id --logging-configuration '{\"LogDestinationConfigs\": [\"arn:partition:logs:region:account-id:log-group:log-group-name\"]}'",
```

Comment:
The logs destination couldn't be a CloudWatch Log Group but a Kinesis stream as per the trendmicro documentation, please update it.
```
aws wafv2 put-logging-configuration
  --region us-east-1
  --logging-configuration ResourceArn=arn:aws:wafv2:us-east-1:123456789012:regional/webacl/cc-malicious-viewers-web-acl/abcdabcd-1234-abcd-1234-abcd1234abcd,LogDestinationConfigs=arn:aws:firehose:us-east-1:123456789012:deliverystream/aws-waf-logs-web-acl-logging
```

Reply:
https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/wafv2/client/put_logging_configuration.html you can see in the function that it could be a CloudWatch Log Group, Kinesis Stream or an S3 bucket.

Reply:
That's right, I've checked WAF instead of WAFv2 I think.